examples/cpp/SampleCppUWP/SampleCppUWP.vcxproj

```diff
@@ -11,6 +11,15 @@
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>

```

Comment:
Maybe need to remove this explicit dependency on particular Win 10 SDK here as well, and just fall-thru to the auto-detection logic. It happens that I do have this one installed on my machine ( `10.0.17763.0` ), as I believe it does come bundled as one of the defaults with the current version of Visual Studio 2019.

Reply:
Yup, I've had to manually update or install SDKs. I don't believe this property is actually needed, at least not in VS2017 and 2019 as I removed and things still built in both.